src/Objs/Homepage/HomepageObjClass.ts

```diff
@@ -20,6 +21,7 @@ export const Homepage = provideObjClass('Homepage', {
     sitePortalOverviewPage: 'reference',
     siteSearchResultsPage: 'reference',
     siteUserProfilePage: 'reference',
+    twitterSite: 'string',
```

Comment:
Please prefix both attributes with `site`, e.g. `siteFacebookAppId` and `siteTwitterSite`.